### PR TITLE
SEO-174775 WindowsForms Page with redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan,

Have worked on changing WindowsForms Page with Redirect UG, please review.

Regards,
Yvone.

### CODE REVIEW

File: WindowsForms/Grid-Control/Undo-Redo.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ documentation: ug
 ---
 # Undo Redo in Windows Forms Grid Control
 
-GridControl gives support for Undo/Redo functionality. GridControl keeps a track on all the operations undergone, in a particular class [GridModelCommandManager](http://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridModelCommandManager.html#). The Undo/Redo operations can be done by using commands, methods and keyboard shortcuts. 
+GridControl gives support for Undo/Redo functionality. GridControl keeps a track on all the operations undergone, in a particular class [GridModelCommandManager](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridModelCommandManager.html#). The Undo/Redo operations can be done by using commands, methods and keyboard shortcuts.
```
Comment (MallikaRK) [inline WindowsForms/Grid-Control/Undo-Redo.md:12]: https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridModelCommandManager.html

why you maintain # symbol is needed?

Comment (MallikaRK) [reply WindowsForms/Grid-Control/Undo-Redo.md:12]: Recheck all the links and update it.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Grid-Control/Undo-Redo.md (hunk 1 above)
Comment (Yvone-Atieno) [reply WindowsForms/Grid-Control/Undo-Redo.md]: updated

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**